civiawards.php

```diff
@@ -161,6 +161,11 @@ function civiawards_civicrm_permission(&$permissions) {
     'CiviAwards: Access applicant portal',
     ts('Allows a user to access the awards applicant portal'),
   ];
+
+  $permissions['create/edit awards'] = [
+    'CiviAwards: Create/edit awards',
```

Comment:
Can we change the label to `CiviAwards: Create/Edit awards` i.e make edit to be sentence case.